### PR TITLE
[Rated Disabilities] Moving the learn and need help sections into their own components

### PR DESCRIPTION
**Pull request by jerekshoe**
## Summary
Moving the learn and need help sections of the `RatedDisabilityView` component into their own components because there is a plan to reuse them in the the components related to the EVSS -> Lighthouse migration

## Related issue(s)
- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#63310

## Screenshots
**Note:** The 'Feedback' button only shows up in staging, the before picture is from staging, the after one is from my local

|               | Before | After |
| -------- | ------ | ----- |
| Desktop | Screenshot 2023-12-19 at 4 11 04 PM | Screenshot 2023-12-19 at 4 22 33 PM |

## What areas of the site does it impact?
Rated Disabilities app

### Authentication
- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/rated-disabilities/components/NeedHelp.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+
+export default function NeedHelp() {
+  return (
+    <va-need-help class="vads-u-margin-y--3">
+      <div slot="content">
+        <p>
+          You can call us at <va-telephone contact={CONTACTS.VA_BENEFITS} />.
```
Comment (pmclaren19) [inline src/applications/rated-disabilities/components/NeedHelp.jsx:9]: This component looks different then what we currently have is this okay? I would also add a screen shot of the before and after

File: src/applications/rated-disabilities/components/Learn.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```
Comment (pmclaren19) [inline src/applications/rated-disabilities/components/Learn.jsx:1]: LGTM!

**Review by pmclaren19 — CHANGES_REQUESTED** [review]
@jerekshoe could you please add a test to RatedDisabilityView.jsx to show that these new components render?

File: src/applications/rated-disabilities/components/NeedHelp.jsx (hunk 1 above)
Comment (jerekshoe) [reply src/applications/rated-disabilities/components/NeedHelp.jsx:9]: Yeah that's intentional. I wanted to make sure we were using the web-component and most of the other places where we mention calling a VA phone number include the 711 sentence I added as well

**Review by pmclaren19 — APPROVED** [review]
Thanks for adding tests! This looks good to me!

**Pull request merged**